consensus/src/experimental/execution_phase.rs

```diff
@@ -21,9 +23,17 @@ use std::sync::Arc;
 pub type ResetAck = ();
 pub fn reset_ack_new() -> ResetAck {}
 
+pub type ExecutionPhaseCallBackType = Option<Box<dyn FnOnce(HashValue, Block) + Send + Sync>>;
```

Comment:
why this is Option?

Reply:
To make the test cleaner.

Reply:
we shouldn't adopt code for test, can't we just create empty callback?

Reply:
Okay, I'll change this a bit later.